integration-tests/test_cdk.py

```diff
@@ -7,11 +7,8 @@
 namespace = os.environ.get('TEST_CHARM_NAMESPACE', 'containers')
 charm_channel = os.environ.get('TEST_CHARM_CHANNEL', 'stable')
 snap_channel = os.environ.get('TEST_SNAP_CHANNEL', '1.8/stable')
-bundles = [
-    'canonical-kubernetes-canal',
-    'kubernetes-core',
-    # 'canonical-kubernetes',
-]
+bundles = os.environ.get('TEST_BUNDLES', 
+                         "'canonical-kubernetes-canal','kubernetes-core'").split(',')
```

Comment:
These strings are double-quoted. Also we should probably strip whitespace after the split, in case someone does:
```
TEST_BUNDLES="canonical-kubernetes-canal, kubernetes-core"
```